src/main/drivers/pwm_output.c

```diff
@@ -48,7 +48,7 @@ static uint16_t freqBeep = 0;
 static bool pwmMotorsEnabled = false;
 static bool isDshot = false;
 #ifdef USE_DSHOT_DMAR
-#ifdef FORCE_DSHOT_DMAR
+#ifdef ENABLE_DSHOT_DMAR
```

Comment:
Why the change here?

Reply:
the rest changed upstream so we changed it to match